cmd/flags_node.go

```diff
@@ -34,31 +32,17 @@ func NodeFlags() *flag.FlagSet {
 		"",
 		"Path to a customized node config TOML file",
 	)
-	flags.String(
-		nodeNetworkFlag,
-		"",
-		"The name of the network to connect to, e.g. "+params.ListProvidedNetworks(),
-	)
 
 	return flags
 }
 
 // ParseNodeFlags parses Node flags from the given cmd and applies values to Env.
 func ParseNodeFlags(ctx context.Context, cmd *cobra.Command) (context.Context, error) {
-	network := cmd.Flag(nodeNetworkFlag).Value.String()
-	if network != "" {
-		err := params.SetDefaultNetwork(params.Network(network))
-		if err != nil {
-			return ctx, err
-		}
-	} else {
-		network = string(params.DefaultNetwork())
-	}
-
+	network := Network(ctx)
```

Comment:
This is more of a maintenance nit [ non-blocking ]: 
the context of why `ParseNodeFlags` must come after `ParseNetwork` (even though it makes sense here) might get forgotten later on, so because getting the network is a requirement for constructing the node store name, maybe `ParseNodeFlags` should take network explicitly as a param?

Reply:
done